.github/workflows/readme-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Check README Updates
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  check-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Find README.docify.md files and check generated READMEs
+        run: |
+          # Find all README.docify.md files
```

Comment:
Can you add the entire bash script under  `.github/scripts/check-missing-readme-generation.sh` or other script name you see fit?